qt/HelloQt.hpp

```diff
@@ -0,0 +1,12 @@
+#include <QWidget>
+
```

Comment:
Some notes on Qt style from their examples:
- Use spaces instead of tabs
- 'public' and 'private slots' should start at column 1, so lines 6-10 should be de-indented by one

Reply:
I really think style guide should be consistent across the cpp_starter_project. If you were going to pick Qt's style guide here, you should consistently use that same style in the other examples.

Better to use the style that is already being used in the rest of the starter project in my opinion.

Reply:
Just run clang-format with your favorite IDE/editor. There's .clang-format file in the top level. There's also a .cmake-format.yml file that can be used to apply appropriate formatting to the CMakeLists.txt